wopiserver/templates/deployment.yaml

```diff
@@ -55,6 +57,9 @@ spec:
                   name: {{ template "wopiserver.fullname" . }}-config
               - secret:
                   name: {{ template "wopiserver.fullname" . }}-secrets
+        - name: {{ .name }}-secret
+          secret:
+            secretName: {{ include "wopiserver.fullname" $ }}-secret
```

Comment:
All secrets are already mounted on the projected volume in L58-59, this is not needed - see comment above.